web/test/pages/onboarding/onboarding-owning.test.tsx

```diff
@@ -0,0 +1,208 @@
+    page.chooseRadio("has-existing-business-true");
+    await page.visitStep2();
+    page.fillText("Entity id", "123");
+    fireEvent.blur(subject.getByLabelText("Entity id"));
+    page.clickNext();
+    await waitFor(() => {
+      expect(subject.getByTestId("step-2")).toBeInTheDocument();
+      expect(
+        subject.getByText(templateEval(OnboardingDefaults.errorTextMinimumNumericField, { length: "10" }))
+      ).toBeInTheDocument();
+      expect(subject.queryByTestId("toast-alert-ERROR")).toBeInTheDocument();
+    });
+    page.fillText("Entity id", "1234567890");
+    await page.visitStep3();
+
+    await waitFor(() => {
+      expect(
+        subject.queryByText(templateEval(OnboardingDefaults.errorTextMinimumNumericField, { length: "10" }))
+      ).not.toBeInTheDocument();
+      expect(subject.getByTestId("step-3")).toBeInTheDocument();
+      expect(subject.queryByTestId("step-2")).not.toBeInTheDocument();
+      expect(subject.queryByTestId("toast-alert-ERROR")).not.toBeInTheDocument();
+    });
+  });
+
+  it("prevents user from moving after Step 3 if you have not entered a business name", async () => {
+    const { subject, page } = renderPage({});
+    page.chooseRadio("has-existing-business-true");
+    await page.visitStep2();
+    await page.visitStep3();
+    page.clickNext();
+    expect(subject.getByTestId("step-3")).toBeInTheDocument();
+    expect(subject.queryByText(OnboardingDefaults.errorTextRequiredBusinessName)).toBeInTheDocument();
+    await waitFor(() => expect(subject.queryByTestId("toast-alert-ERROR")).toBeInTheDocument());
+    page.fillText("Business name", "A business");
+    page.clickNext();
+
+    await waitFor(() => {
+      expect(subject.queryByText(OnboardingDefaults.errorTextRequiredBusinessName)).not.toBeInTheDocument();
+      expect(subject.queryByTestId("toast-alert-ERROR")).not.toBeInTheDocument();
+    });
+  });
+
+  it("prevents user from moving after Step 4 if you have not selected a location", async () => {
+    const newark = generateMunicipality({ displayName: "Newark" });
+    const { subject, page } = renderPage({ municipalities: [newark] });
+    page.chooseRadio("has-existing-business-true");
+    await page.visitStep2();
+    await page.visitStep3();
+    page.fillText("Business name", "A business");
+    await page.visitStep4();
+    page.clickNext();
+    await waitFor(() => {
+      expect(subject.getByTestId("step-4")).toBeInTheDocument();
+      expect(subject.queryByText(OnboardingDefaults.errorTextRequiredMunicipality)).toBeInTheDocument();
+      expect(subject.queryByTestId("toast-alert-ERROR")).toBeInTheDocument();
+    });
+    page.selectByText("Location", "Newark");
+    page.clickNext();
+    await waitFor(() => {
+      expect(subject.queryByText(OnboardingDefaults.errorTextRequiredMunicipality)).not.toBeInTheDocument();
+      expect(subject.queryByTestId("toast-alert-ERROR")).not.toBeInTheDocument();
+    });
+  });
```

Comment:
In all of these tests you just check for the error validation, shouldn't we also be checking to make sure it's still not routing you to the next task?